crates/hir-def/src/macro_expansion_tests/builtin_fn_macro.rs

```diff
@@ -143,7 +143,7 @@ macro_rules! assert {
 
 fn main() {
      {
-        if !true {
+        if !(true ) {
```

Comment:
Does anything warn about this?

Reply:
I imagine our expansions are losing the delimiter information here? (I remember us having problems with the subtree delimiters)

Reply:
I feel like I saw the same change somewhere else. Oh well, bo idea.

Reply:
I just changed `assert` macro expansion to make `assert!(2 + 2 == 4)` working (it was expanded to `if !2 + 2 == 4`). It's otherwise unrelated to this PR.